easybuild/easyblocks/generic/intelbase.py

```diff
@@ -97,8 +94,8 @@ def __init__(self, *args, **kwargs):
         self.setup_local_home_subdir()
         self.clean_home_subdir()
 
-        if self.cfg._config['license'][0] is not None:
-            _log.deprecated('No old style license parameter, use license_file', '2.0')
+        if 'license' in self.cfg._config:
```

Comment:
even better: just drop this, since this check is already done in framework (@ https://github.com/hpcugent/easybuild-framework/blob/develop/easybuild/framework/easyconfig/easyconfig.py#L264)

if `license` is defined in the easyconfig, the `EasyConfig` instance will never be created since it'll die in `parse()`, so you'll never even get here